### DISCUSSION TITLE
How to be able to use 2 different clients with the same "Username"

### DISCUSSION THREAD

**Discussion by dredhorse** [Pryaxis/TShock#2081] [Q&A] [answered]
Hi,

I did just setup a tshock server and logged out. After that I switched computers and did login to the server again from an other Computer.

I can't login now as the User as my UUIDs don't match, which is valid of course.

But how can I allow playing from two different clients with different UUIDs and the same Username?

**Comment by dredhorse** [comment, marked as answer]
Answer: just do /login UserName password and this will take over the account

**Reply by Quinci135** [reply]
You could also copy your uuid (located in your config.json) from your logged in computer to the other and both computers can automatically login to your account, without the need to /login every time.

**Reply by dredhorse** [reply]
good point for other players!

Doesn't work for me because I play on macOS Catalina via geforce now and on a Windows 10. (Which makes me wonder when Terraria is updated to 64bit 😞 )
